### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by SimaMN**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Add the code for assignment 1.
## What did you learn from the changes you have made?
I learned that incorporating Batch Normalization and Dropout—can significantly improve training stability and reduce overfitting. 
## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
n/a
## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
n/a
## How were these changes tested?
Ran the code in a jupiter notebook.
## A reference to a related issue in your repository (if applicable)
n/a
## Checklist
- [X] I can confirm that my changes are working as intended

### CODE REVIEW

**Review by vishnouvina — APPROVED** [review]
A1 approved
